x/farming/keeper/reward.go

```diff
@@ -370,7 +375,15 @@ func (k Keeper) AllocationInfos(ctx sdk.Context) []AllocationInfo {
 	// In this step, we check if farming pools have sufficient balance for allocations.
 	// If not, we don't allocate rewards from that farming pool for this epoch.
 	var allocInfos []AllocationInfo
-	for farmingPool, planCoins := range allocCoins {
+	// Sort map keys for deterministic execution.
+	var farmingPools []string
+	for farmingPool := range allocCoins {
+		farmingPools = append(farmingPools, farmingPool)
+	}
+	sort.Strings(farmingPools)
```

Comment:
This is me nitpicking to have space between two different logics. For better readability, I suggest to have space in the following line. Also, for the same reason I think `var allocInfos []AllocationInfo` can move down to line 385-386 (after `sort.Strings(farmingPools)`).
```suggestion
	sort.Strings(farmingPools)
	
```

Reply:
I updated the code style, thanks.